File: CSharp/HtmlWalker/Exception/InvalidVisitorException.cs
namespace HtmlWalker.Exception
{
    /// <summary>
    ///   This exception is called when a visitor has been given to
    ///   a <see cref="Walker"/> that does not meet it's specifications
    ///
    ///   See the <b>Walker</b> specification.
    /// </summary>
    /// <author>Robert R Flores</author>
    internal class InvalidVisitorException : HtmlWalkerException
    {
        public InvalidVisitorException() : base() { }

        public InvalidVisitorException(string msg) : base(msg) { }
    }
}

File: Java/htmlwalker/src/main/java/htmlwalker/exception/InvalidVisitorException.java
package htmlwalker.exception;

@SuppressWarnings("serial")
public class InvalidVisitorException extends HtmlWalkerException
{

	public InvalidVisitorException() { super(new IllegalStateException()); }
	
	public InvalidVisitorException(String msg) { super(new IllegalStateException(msg)); }
	
}
